schemaless/create_uuid_map.py

```diff
@@ -17,25 +18,45 @@ def __init__(self):
     def from_files(cls, schemaless_file, uuid_map_file):
         """Build a RecordGraph from schemaless & uuid_map files."""
         rg = cls()
+
+        latest_records = latest_values(schemaless_file)
+
+        # Create a mapping between permit numbers and their associated PPTS
+        # record (so we an ensure they are assigned the same UUID).
+        permit_number_to_ppts = {}
+        for fk, record in latest_records[PPTS.NAME].items():
+            if record['related_building_permits']:
+                for permit_number in record['related_building_permit'].split(","):
+                    #  This assumes that a given permit can only have one PPTS
+                    #  parent. Is that accurate? 
+                    permit_number_to_ppts[permit_number] = fk
+
         # Read the latest values from the schemaless file to build the graph.
-        for fk, record in latest_values(schemaless_file).items():
-            parents = []
-            children = []
-            if record['parents']:
-                parents = record['parents'].split(",")
-            if record['children']:
-                children = record['children'].split(",")
-            the_date = None
-            for source in source_map.values():
-                if source.DATE_KEY in record:
-                    the_date = source.get_date(record)
-            rg.add(Node(
-                record_id=fk,
-                date=the_date,
-                parents=parents,
-                children=children,
-                uuid=None,
-            ))
+        for source, source_records in latest_records.items():
+            for fk, record in source_records.items():
+                parents = []
+                children = []
+                if source == PPTS.NAME:
```

Comment:
I'm going to try moving this logic to the `Source` objects when I do the same thing for TCO data

Reply:
sgtm, thanks!